js/scripts.js

```diff
@@ -0,0 +1,22 @@
+(function($, window, document) {
+    $(function() {
+
+        parallax();
+        $( window ).resize(function() {
+            parallax();    
```

Comment:
is there logic in the parallax plugin to not recreate it every time? This could be a somewhat expensive call on older computers

Reply:
No that i've seen so far.

We can decide to change the parallax plugin if needed, I decided to go with this one since its easy to use and so i can continue implementing the design.

Reply:
we could always add a boolean and check if we've enabled it already? Just an idea to avoid multiple instances of parallax potentially running